gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -983,25 +990,17 @@ def plug_interfaces_fast(self, event):
         orchestration_driver = self._get_orchestration_driver(
             service_details['service_vendor'])
 
-        ports = []
-        if consumer['port']:
-            ports.append(
-                {'id': consumer['port']['id'],
-                 'port_classification': consumer['port_classification'],
-                 'port_model': consumer['port_model']})
-        if provider['port']:
-            ports.append(
-                {'id': provider['port']['id'],
-                 'port_classification': provider['port_classification'],
-                 'port_model': provider['port_model']})
+        ports = self._make_ports_dict(consumer, provider, 'port')
```

Comment:
why call again here ? self._make_ports_dict is already called above, 'ports' are assigned above.